### PR TITLE
fix: added an warning about the new input sytem changes to the xr setup

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
### Description
Added the warning for oculus and windows MR, not sure this is correct @Gusinuhe ?

### CODE REVIEW

File: Editor/UI/Wizard/Setup/XRSDKSetupPage.cs [hunk 1]
```diff
@@ -67,6 +67,11 @@ public override void Draw(Rect window)
                         otherHardwareText = CreatorGUILayout.DrawTextField(otherHardwareText, -1,GUILayout.Width(window.width * 0.4f));
                     GUILayout.EndHorizontal();
                 }
+
+                if (selectedLoader == XRLoader.Oculus || selectedLoader == XRLoader.WindowsMR)
+                {
+                    EditorGUILayout.HelpBox("This selection will install Unity XR Toolkit, which includes Unity's new InputSystem. By installing this, the event handling will be switched, which disables UIs created with IMGUI.\n\nTo fix this go to 'ProjectSettings/Player/Other Settings' and set 'Active Input Handling' to 'Both'", MessageType.Warning);
+                }
```
Comment (Gusinuhe) [inline Editor/UI/Wizard/Setup/XRSDKSetupPage.cs:74]: Actually, at the moment we show this warning, XR Interaction Toolkit is already installed, the message has no sense and can be hidden for users that either already has an XR configuration or select something else but Oculus and WindowsMR 😱

**Review by Gusinuhe — CHANGES_REQUESTED** [review]

**Comment by Gusinuhe** [comment]
This was not supposed to be merged :(

**Comment by Gusinuhe** [comment]
This issue is already addressed in https://github.com/Innoactive/XR-Interaction-Component/pull/73, I suggest to close this PR.

**Comment by SimonTheSourcerer** [comment]
So we close it?

**Pull request closed without merging**